docs/source/install.rst

```diff
@@ -32,6 +32,60 @@ Install from Source
     $ pip install git+https://github.com/pydata/pandas-gbq.git
 
 
+Install in Development Mode
+---------------------------
+
+Follow these instructions to install pandas-gbq if you want to make changes to
+the library.
+
+`Download the code <https://github.com/pydata/pandas-gbq/archive/master.zip>`__
+from GitHub or clone the git repository.
+
+.. code-block:: shell
+
+    $ git clone https://github.com/pydata/pandas-gbq.git
+    $ cd pandas-gbq
+
+Pip
+~~~
+
+To install pandas-gbq and its dependencies in development mode via pip
+
+.. code-block:: shell
+
+    $ pip install -e .
```

Comment:
we generally don't use pip to install things when we have proper conda packages. sure you can, but its not the first thing you do.